pycloudlib/oci/cloud.py

```diff
@@ -204,13 +204,13 @@ def launch(self, image_id, instance_type='VM.Standard2.1', user_data=None,
 
         instance_data = self.compute_client.launch_instance(
             instance_details).data
-        instance_data = wait_till_ready(
-            func=self.compute_client.get_instance,
-            current_data=instance_data,
-            desired_state='RUNNING',
-        )
         instance = self.get_instance(instance_data.id)
```

Comment:
If we haven't waiting for the launch call, will an `id` be available if the launch hasn't completed?